http-add-on/templates/routing-table-config-map.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: keda-http-routing-table
+  labels:
+    "control-plane": "operator"
+    "keda.sh/addon": "http-add-on"
+    "app":           "http-add-on"
+    "name":          "http-add-on-routing-table"
+data:
```

Comment:
On purpose as a place holder or?

Reply:
removed in dbd741343de44b40b1b7cc1957aaa0d239a6dc0a